equinox/nn/_batch_norm.py

```diff
@@ -154,23 +180,35 @@ def _stats(y):
                 var = jnp.maximum(0.0, var)
                 return mean, var
 
-            first_time = state.get(self.first_time_index)
-            state = state.set(self.first_time_index, jnp.array(False))
+            momentum = self.momentum
+            zero_frac = state.get(self.zero_frac_index)
+            zero_frac *= momentum
```

Comment:
Stylistic nit: I tend not to use the inplace operations in JAX code. This (a) fits with the functional style a bit better, and (b) emphasises that we're definitely falling back to the `zero_frac = zero_frac * momentum` interpretation of the syntax. (Gosh, why does Python has two different meanings for the same syntax?)

Reply:
makes sense, done